src/views/process-panel/information-card.tsx

```diff
@@ -0,0 +1,132 @@
+        width: theme.spacing.unit * 12,
+        backgroundColor: theme.customs.colors.green700,
+        color: theme.palette.common.white,
+        fontSize: '0.875rem',
+        borderRadius: theme.spacing.unit * 0.625
+    },
+    headerText: {
+        fontSize: '0.875rem',
+        position: 'relative',
+        top: -theme.spacing.unit * 4.5,
+        left: theme.spacing.unit * 3,
+    }
+});
+
+interface ProcessInformationCardDataProps {
+    item: ProcessResource;
+}
+
+type InformationCardProps = ProcessInformationCardDataProps & DispatchProp & WithStyles<CssRules>;
+
+export const InformationCard = withStyles(styles)(
+    connect((state: RootState) => ({
+        item: state.collectionPanel.item
+    }))(
+        class extends React.Component<InformationCardProps> {
+            render() {
+                const { classes } = this.props;
+
+                return <div>
+                    <Card className={classes.card}>
+                        <CardHeader
+                            avatar={<ProcessIcon className={classes.iconHeader} />}
+                            action={
+                                <IconButton
+                                    aria-label="More options"
+                                    onClick={this.handleContextMenu}>
+                                    <MoreOptionsIcon />
+                                </IconButton>
+                            }
+                            title="Pipeline template that generates a config file from a template" />
+                        <CardContent className={classes.content}>
+                            <Grid container direction="column">
+                                <Grid item xs={8}>
+                                    <DetailsAttribute classLabel={classes.label} classValue={classes.value}
+                                        label='Status' value={<Chip label="Complete" className={classes.chip} />} />
+                                    <DetailsAttribute classLabel={classes.label} classValue={classes.value}
+                                        label='Started at' value="1:25 PM 3/23/2018" />
+                                    <DetailsAttribute classLabel={classes.label} classValue={classes.value}
+                                        label='Finished at' value='1:25 PM 3/23/2018' />
+                                </Grid>
+                            </Grid>
+                            <Grid container direction="column">
+                                <Grid item xs={8}>
+                                    <DetailsAttribute classLabel={classes.link} classValue={classes.value}
+                                        label='Container output' />
+                                    <DetailsAttribute classLabel={classes.link} classValue={classes.value}
+                                        label='Show inputs' />
+                                    <DetailsAttribute classLabel={classes.link} classValue={classes.value}
```

Comment:
Add attr `cursor: 'pointer'` to the `DetailsAttribute` core (the class responsible for link).